src/main/java/edu/ucla/library/bucketeer/utils/JobFactory.java

```diff
@@ -26,6 +26,7 @@
 /**
  * A creator of jobs.
  */
+@SuppressWarnings("PMD.NonThreadSafeSingleton") // #FIXME but ignoring for now since
```

Comment:
since.... ?

Reply:
I'd be OK with just "possible area for improvement" if you want to keep it vague

Reply:
Oh "since it's not related to this ticket"... it was something that running the code through PMD highlighted. PMD was turned on thanks to updating to the most recent version of the parent project, but that caused code changes not specifically related to solving the bug in hand.

Reply:
I'll push up the rest of that sentence :-)